.github/ISSUE_TEMPLATE/plugin-request-for-buf-schema-registry.md

```diff
@@ -42,6 +38,8 @@ https://github.com/protocolbuffers/protobuf-go/tree/master/cmd/protoc-gen-go
 
 Unsure? Let's chat!
 
-Not ready to open an issue, but want to chat about your plugin. Come find us on our Public Slack channel
+---
+
+Not ready to open an issue, but want to chat about your plugin? Come find us on our Public Slack channel:
 
-🔗 [https://buf.build/links/slack](https://buf.build/links/slack)
+https://buf.build/links/slack
```

Comment:
I actually wonder if this belongs more at the top of the template, as it's somewhat buried down here?

Separately, removed the markdown link, because the intended person to click the link is the person filling out the template (which is rendered in plaintext); before submitting the issue they'll likely remove the link altogether. I can change it back, though.

Reply:
cc @mfridman, curious what you think.

Reply:
Moving this up towards the top make sense to me. I trust your judgement, we can iterate on this as we go and gather feedback.